hbase-client/src/main/java/org/apache/hadoop/hbase/HTableDescriptor.java

```diff
@@ -857,19 +863,31 @@ public void removeCoprocessor(String className) {
   public static final HTableDescriptor NAMESPACE_TABLEDESC
     = new HTableDescriptor(TableDescriptorBuilder.NAMESPACE_TABLEDESC);
 
+  /**
+   * @deprecated since 0.94.1
```

Comment:
I'm assuming there will be a follow-up task to remove anything that can be removed?

Reply:
Yes. This ticket is based on a discussion we had on the dev list. We have some deprecations where it isn't documented in the code in what release the appropriate functionality was deprecated. There are already some removals in progress, because removing everything at once will end up in a mess.